control-plane/stor-port/src/types/v0/store/volume.rs

```diff
@@ -18,7 +18,20 @@ use crate::{
 use pstor::ApiVersion;
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
+use strum_macros::{EnumCount as EnumCountMacro, EnumIter};
+/// Volume properties.
+#[derive(Debug, EnumCountMacro, EnumIter)]
+pub enum VolumeAttr {
+    MaxSnapshots,
```

Comment:
doc comment this too

Reply:
done

Reply:
hmm don't see the doc comment for MaxSnapshots?